src/components/PlaylistPageCard/PlaylistPageCard.css

```diff
@@ -0,0 +1,23 @@
+.card-overlay-content{
+    color: var(--clr-white);
+    justify-content: center;
+    align-items: flex-end;
+}
+.content{
```

Comment:
I think it is nice to maintain at least a line space between two declarations.

Reply:
actually its my code formatter, so it happens everytime